moped-editor/src/auth/user.js

```diff
@@ -110,11 +110,10 @@ export const initializeUserDBObject = (userObject) => {
       // Then we parse the response
       res.json().then((resData) => {
         if (resData?.errors) {
-          console.error(resData.errors)
+          console.error(resData.errors);
         }
         if (resData?.data?.moped_users) {
           setSessionDatabaseData(resData.data.moped_users[0]);
-          window.location.reload();
```

Comment:
this lil guy is the reason the Moped app loads then a few seconds later reloads when you first login. and it's also causing an extra activity event to be inserted on login.

@mddilley spotted this, and we have both traced the data flow and think it can be safely removed.